apps/dotcom/scripts/build.ts

```diff
@@ -27,6 +27,12 @@ function loadSpaRoutes() {
 		check: true,
 		src: route.vercelRouterPattern,
 		dest: '/index.html',
+		headers: {
+			'Strict-Transport-Security': 'max-age=63072000; includeSubDomains; preload',
+			'X-Content-Type-Options': 'nosniff',
+			'Referrer-Policy': 'no-referrer-when-downgrade',
+			'Content-Security-Policy': 'default-src: *',
+		},
```

Comment:
would we not need to add any/all of these to our assets/api/index routes below?

Reply:
good question.
- we don't need them for the API routes.
- for assets, the `X-Content-Type-Options` is useful to add - so i've added it.
- good point on the 404 page, i've added it there too 👍